templates/web/base/waste/bulky/intro.html

```diff
@@ -9,15 +9,18 @@ <h3>Before you start your booking</h3>
     <li>Requesting a [% service_name %] collection usually takes approximately <strong>10 minutes</strong></li>
```

Comment:
Requesting a small items collection usually takes around 10 minutes

Reply:
Will make this general change.